src/Core/DynamicMetadata/SubjectArea.php

```diff
@@ -32,7 +32,7 @@ class SubjectArea implements DynamicMetadataInterface
      * @param int $x      X-point
      * @param int $y      Y-point
      */
-    public function __construct($x, $y, $width, $height)
+    public function __construct($x, $y, $width = 1, $height = 1)
```

Comment:
Fixed PHPDoc, and added the preliminary validation of the values, throwing an InvalidArgumentException